living_documentation_generator/github_projects.py

```diff
@@ -6,50 +28,53 @@
 
 from living_documentation_generator.model.github_project import GithubProject
 from living_documentation_generator.model.project_issue import ProjectIssue
-from living_documentation_generator.utils.github_project_queries import GithubProjectQueries
+from living_documentation_generator.utils.github_project_queries import (
+    GithubProjectQueries,
+)
 
 logger = logging.getLogger(__name__)
 
 
 class GithubProjects:
+    """
+    A class representing all the logic for mining data for GitHub Projects.
 
+    Attributes:
+        __token (str): The GitHub token used for authentication.
+        __session (requests.Session): The session for making graphQL requests.
+    """
     def __init__(self, token: str):
         self.__token = token
         self.__session = None
 
-    def __initialize_request_session(self):
+    def __initialize_request_session(self) -> requests.Session:
         """
         Initializes the request Session and updates the headers.
-
-        @param github_token: The GitHub user token for authentication.
-
-        @return: A configured request session.
         """
 
         self.__session = requests.Session()
         headers = {
             "Authorization": f"Bearer {self.__token}",
-            "User-Agent": "IssueFetcher/1.0"
+            "User-Agent": "IssueFetcher/1.0",
         }
         self.__session.headers.update(headers)
 
         return self.__session
 
     def __send_graphql_query(self, query: str) -> Optional[dict[str, dict]]:
         """
-        Sends a GraphQL query to the GitHub API and returns the response.
-        If an HTTP error occurs, it prints the error and returns an empty dictionary.
-
-        @param query: The GraphQL query to be sent in f string format.
+        Send a GraphQL query to the GitHub API and returns the response.
+        If an HTTP error occurs, it prints the error and returns None instead.
 
-        @return: The response from the GitHub GraphQL API in a dictionary format.
+        Args:
+            query (str): The formated GraphQL query to send to the GitHub API.
```

Comment:
We are missing info about returned values in this format.

Reply:
All docstrings updated in the commit: [d5e2d21](https://github.com/AbsaOSS/living-doc-generator/pull/30/commits/d5e2d21d6a8e7aa87c1fec62ec1f7e51b477b421)